mne/preprocessing/xdawn.py

```diff
@@ -16,7 +16,7 @@
 from ..io.pick import _pick_data_channels
 from ..utils import logger
 from ..externals.six import iteritems, itervalues
-
+from ..stats.regression import _prepare_rerp_data, _prepare_rerp_preds, _solver
```

Comment:
ok, if it's not numerically the same, I would add a 'solver' string param to Xdawn, but keep it to pinv by default to facilitate the deprecation cycle.

The XdawnTransformer could be using the cholevsky by default.